Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -159,6 +154,50 @@ class BTHTTP: NSObject, URLSessionTaskDelegate {
             return
         }
     }
+    
+    
+    func httpRequest(
+        method: String,
+        path: String,
+        configuration: BTConfiguration? = nil,
+        parameters: Encodable? = nil,
+        headers: [String: String]? = nil
+    ) async throws -> (BTJSON?, HTTPURLResponse?) {
+        let request = try createRequest(
+            method: method,
+            path: path,
+            configuration: configuration,
+            parameters: parameters,
+            headers: headers
+        )
+        let (data, response) = try await session.data(for: request)
+        guard let httpResponse = createHTTPResponse(response: response) else {
+            throw BTHTTPError.httpResponseInvalid
+        }
+        guard let data = data as Data? else {
+            throw BTHTTPError.dataNotFound
+        }
+        if httpResponse.statusCode >= 400 {
+            var error: Error = BTHTTPError.clientError([:])
+            handleHTTPResponseError(response: httpResponse, data: data) { _, err in
```

Comment:
I think we tend to prefer using the fully named params. Are we able to update this to `error` or something more explicit instead of `err`?